websites/D/Dek-d/presence.ts

```diff
@@ -0,0 +1,312 @@
+const presence = new Presence({
+		clientId: "1314484095614451822",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/9wQMfAA.png",
```

Comment:
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).